<!DOCTYPE html>
<html>
	<head>
  	<link rel="stylesheet" href="stylesheet.css">
		<script src="functions.js"></script>

	</head>
	<body>

		<div id="top" class="header">
		<h1>Djuna von Maydell</h1>
		<img src="./MIT-logo-ltgray-white-72x38.svg" >
		</div>

		<div  class="body-sec2">
		<h3>PhD student in the Tsai and Kellis labs, interested in using genomics and computational tools to unravel Alzheimer's disease heterogeneity and polygenic risk on the single-cell level.</h3>
		<p><a href="#current"> current work </a></p>
		<p><a href="#labs">  at these labs </a></p>
		<p><a href="#previous">  previous work </a></p>
		<p><a href="#writing">  writing </a></p>
		<p><a href="#cv">   read my CV </a></p>
		<p><a href="#teaching">  teaching  </a></p>
		<p><a href="#cycling">  MIT cycling team </a></p>
		<p><a href="#contact">  contact me </a></p>

		</div>

		<div id="current" class="body-sec3">
		<h4 class="alignleft">current work</h4>
		<p class="alignright"><a href="#top"> back to top </a></p>
		<div style="clear: both;"></div>
	</div>
	<div id="current" class="body-sec">
		<h3> Questions I am currently thinking about... </h3>

		<p > 1. Can we identify key nodes in gene networks and specific cell-types, onto which multiple Alzheimer's disease (AD) genetic and environmental risk factors converge? </p>
		<p > Multiple genetic and environmental/lifestyle risk factors are known to increase the risk of Alzheimer's disease. To result in neurodegeneration, these factors must exert a cellular and/or biochemical effect, resulting in the accumulation of toxic protein products, and an insufficient protective cellular response, with the ultimate consequence of neuronal death and brain atrophy. Given the diversity of genetic and environmental risk factors, an interesting and obvious question is - At what point do these factors converge? By shifting cell states of particular cell types? By activating certain genes? Or do they meet all the way downstream - and converge on neuronal death? </p>
		<p > 2. How do multiple AD genetic risk variants interact to increase risk in AD? </p>
		<p >Current studies that dissect genetic mechanisms of Alzheimer's disease have focused largely on the isolated effect of individual genetic risk variants, when controling for multiple covariates. Now that genome wide association studies have implicated ~ 29 genes in increasing disease risk, an interesting consideration is - What happens when multiple variants converge in a single individual? How does this modify risk, i.e. How do multiple genetic risk variants alter cell states that promote or protect against AD pathology, neuroinflammation, and neurodegeneration? This is an important question, as beginning to answer it will elucidate novel mechanisms of disease pathogenesis, which might explain the observed transcriptomic and clinical disease heterogeneity observed in this disease.</p>
		<p> 3. How can we classify individuals and generate a hierarchy of AD molecular subtypes to dissect AD heterogeneity? </p>
		<p > AD is a heterogenous disease, on the molecular and the clinical level. I strongly believe that we should redefine our understanding of AD, not as a single disease, but as a large group of distinct molecular processes initiated by diverse upstream risk factors that converge to the common phenotypes of amyloid deposition, tangle formation, and dementia. I am interested in using high-dimensional molecular signatures to identify functional subtypes of AD and systematically characterize heterogeneity of the disease. A systematic and quantitative classification of AD subtypes, akin to the classifications that inform therapeutic targets in cancer, will aid therapeutic intervention, biomarker identification of increased sensitivity, as well as systematic stratification of cohorts in scientific studies. </p>
		<p > 4. How can we infer spatial localization from single-cell RNA-seq data? </p>
		<p > This is another question I am very interested in. One limitation with single-cell RNA-sequencing technology is that it removes cells from their native environment, and therefore loses the spatial component of variation that defines tissue architecture. Multiple studies have shown that low-throughput spatial transcriptomic data can be integrated with single-cell RNA-sequencing data to infer a pairwise distance matrix from the single-cell RNA-sequencing dataset - thus restoring the spatial component that was lost by single-cell transcriptomics. However, a major drawback here is that these methods require scientists to generate paired spatial and single-cell data, which is not always experimentally feasible. I am interested in developing probabilistic methods to infer spatial information from single-cell data alone.</p>
		<p >These are the questions I am currently trying to tackle during my PhD. </p>

		<h3> Guiding interests... </h3>
		<p > 1. Genetics and epigenetics: This discipline has fascinated me since high-school. To me, this is one of the most fascinating fields in biology, which continues to change rapidly and requires sophisticated computational methods to deal with the high-dimensional datasets generated in genomics.</p>
		<p > 2. Single-cell RNA-sequencing technologies: This is an absolutely amazing and fascinating technology. It allows us to identify unknown celltypes and cell states in an unsupervised way. It also allows us to investigate cell-type heterogeneity within a single individual, which opens all kinds of possibilities (for example, now we can now use machine learning to classify cells within a single subject according to how pathogenic they are, how close to pathology we infer their spatial location, whether there is a subpopulation of cells, whose allelic expression is skewed... and many more interesting questions and applications.)</p>
		<p > 3. Protein structure and function: This is slighly out of the scope of the previous interests - But protein structure and function has always fascinated me. It's an important consideration for Alzheimer's disease too - previous research suggests that genes that are toxic upon overexpression (i.e. dosage sensitive) tend to have more intrinsically disordered regions - they might enter into promiscuous interactions in the cell and are more likley to be hubs that interact with (and thus dysregulate) multiple proteins upon overexpression <a href=https://www.cell.com/cell/pdf/S0092-8674(09)00454-1.pdf >(click here for reference)</a>. I find this biochemical view on Alzheimer's disease very compelling.</p>

		</div>

		<div id="labs" class="body-sec3">
		<h4 class="alignleft">at these labs</h4>
		<p class="alignright"><a href="#top"> back to top </a></p>
		<div style="clear: both;"></div>
	<div  class="body-sec2">

	<h3>I am part of the Tsai lab (advised by Li-Huei Tsai) in the Brain and Cognitive Sciences Department and the Kellis lab (advised by Manolis Kellis) in the Computer Science department.</h3>
		<h3>Tsai lab @ MIT</h3>
		<p>The Tsai lab uses a multidisciplinary approach to understand the mechanisms that cause neurodegeneration. We investigate circuit mechanisms of degeneration, epigenetic signatures of degeneration, and study the genetic risk that causes Alzheimer's disease. We use a range of technologies to achieve this: mouse models, in vitro models using human-derived induced-pluripotent stem cells, organoids, microfluidic devices, and human brain samples. All of these are sources of high dimensional datasets, including epigenomic data, transcriptomic data, and genomic sequences. Technological advances allow us to place a particular emphasis on high-dimensional datasets derived from single cells. </p>
		<p><a href="https://tsailaboratory.mit.edu/publications/"> Check out some of the Tsai lab's work</a></p>
		<h3>Kellis lab @ MIT</h3>
		<p>The Kellis lab develops new computational methods and analyzes high-dimensional biological datasets. We receive a subset of the exciting datasets generated in the Tsai lab and analyze these using new and existing computational methods. For example, to decompose single-cell data into low dimensional space, I use a method called "ACTIONet", developed by the Kellis lab. This method attempts to represent the transcriptional profile in a small number of components, assumed to optimally represent the transcriptional variability in the dataset, based on which cell neighborhoods are inferred.  <p>
		<p><a href="http://compbio.mit.edu/papers.html"> Check out some of the Kellis lab's work</a></p>
		</div>


		<div id="previous" class="body-sec3">
		<h4 class="alignleft">previous work</h4>
		<p class="alignright"><a href="#top"> back to top </a></p>
		<div style="clear: both;"></div>
<div  class="body-sec">
	<h3>Here are descriptions of some research projects I have previously worked on...</h3>

		<p>1. Dissecting the Role of Individual Aβ Species on AD Pathogenesis in 3D Cultured Human Neurons </p>

		<p>Aß42 is thought to be a key mediator of pathogenesis in AD. However, the impact of Aß42 versus Aß40 and other
		Aß isoforms has not been investigated in human cellular models. During my time in the Tanzi lab at Harvard Medical School, I helped engineer a novel 3D cell culture model of AD to dissect the contribution of individual Aß species to disease pathogenesis. Based on recent insights into the sequential cleavage cascade of Amyloid Precursor Protein (APP), we found that insertion of aromatic residues at select APP transmembrane domain (TMD) positions modulates the Aß-42/40 ratio in cell lines over-expressing APP.
		hNPCs with various APP TMD mutations differentiated in 3D provide a novel platform with which to characterize
		the role of individual Aß species in AD pathogenesis.  </p>

		<p>The guiding biochemical logic: Ɣ-secretase processes the transmembrane domain (TMD) of APP via two main pathways
		of sequential tripeptide cleavage. The enzyme initiates cleavage after
		Leu49 or Thr48, which results in the production of Aβ-40 or -42, respectively (A). The structure
		of the 3 S' pockets in the presenilin catalytic subunit can be exploited to direct cleavage
		towards either Aβ isoform. Recent research elucidates the structural basis of ɣ-secretase’s
		tripeptide cleavage mechanism; The enzyme’s S'2 pocket is small and does not
		accommodate large residue side-chains. Insertion of aromatic residues at TMD positions 50
		or 51 (V50F and M51W) decreases or increases the Aβ-42/40 ratio, respectively, by
		decreasing favorable interactions with the initial cleavage site of either cleavage pathway
		(B). Based on these findings, we  introduced select
		mutations into the APP TMD and showed that these mutations did indeed modulate Aβ-42/40 ratios in cell lines transfected with
		these constructs. This provides a model to enable study of the Abeta isoforms, as well as drug screening to modulate their ratios. This work was published as part of a study in Nature Communications, 2020.  <a href="https://www.nature.com/articles/s41467-020-15120-3"> View the paper</a></p>
		<img src="./f1.svg" >

		<p>2. Analysis of transcriptomic signature of individual Aβ isoforms: </p>

		<p> To identify gene changes associated with Aβ toxicity, I characterized the transcriptomic response of differentiated 3D-cultured AD human neurons to Aβ accumulation. The cartoon below captures the workflow for this project: a Familial AD hNPCs were generated by transfection with pathogenic (CSCW-APPSL/I45F-(PS1Δe9)-GFP/mCherry) or control (CSCW-GFP) constructs. Mutant APP and GFP-marker expression are controlled by the Cytomegalovirus promoter (CMV). Internal Ribosomal Entry Site (IRES) allows for separate translation of APP and GFP. This is followed by enriching cells clonally using FACS. Clonal cell lines were expanded in 2D cell culture flasks. Clonal cell lines were then differentiated within an artificial extracellular matrix in 3D, which is accompanied by a robust generation of AD pathologies including amyloid plaques and tau tangles. b After differentiation for 7- and 4-weeks, cells were harvested, and mRNA extracted. cDNA libraries were prepared from the mRNA samples (N=3 per cell line) and sequenced on an Illumina platform. Reads were aligned to the human genome and differential gene expression analysis performed between control and FAD cell lines. Differentially expressed genes were analyzed with various methods. c Gene sets from commonly enriched pathways were analyzed for differential enrichment in control versus AD transcriptomes from human temporal cortex and cerebellum using gene set enrichment analysis.  </p>
		<img src="./f2.png" >

		<p>3. Identifying mechanisms and biomarkers of spontaneous preterm birth</p>

		<p>Complications following spontaneous preterm birth (sPTB) constitute a major cause of neonatal death worldwide.
		Biomarkers which are predictive of sPTB early in gestation are currently unavailable, hindering effective intervention.
		I worked in Dr. Mason’s research group to conduct research for my undergraduate honors thesis, titled
		“Stratification of Cervico- Vaginal 1H NMR Metabolomics Data Reveals Biomarkers of Spontaneous Preterm Birth”.
		I analyzed vaginal microbiome and metabolome data, to identify biomarkers of
		spontaneous preterm birth (sPTB) and gain insight into potential mechanisms of sPTB. These data were generated
		in the context of the INSIGHT prospective cohort study, led by chief investigators Dr. Rachel Tribe and Professor
		Andrew Shennan.
		I evaluated to what extent certain
		metadata variables, such as ethnicity, the microbiome, disease, and risk stratification, were correlated with
		variation in the vaginal metabolome. To control for confounding and test for potential effect modification, I
		investigated whether data stratification according to select metadata variables, would reveal biomarkers of sPTB,
		which are specific for certain patient cohorts. Finally, I integrated metabolome data, microbiome data, and other
		variables associated with sPTB, to evaluate the importance of these features in
		predicting sPTB in the entire and stratified cohorts. We identified multiple features that were predictive of preterm birth in data stratified according to selct metadata variables. Hierarchical stratification, using multiple biomarkers, may increase the specificity of predictive sPTB tests for
		clinical use.</p>

		<p>Working with these human datasets (at the time, working with patient data was a first for me) exposed me to multiple variables I had not dealt with previously. One of these was "ethnicity". This had me thinking: How is ethnicity defined? How are ethnicities grouped together? Based on cultural and social factors, genetics, or racist beliefs? I found a lot of interesting answers to my questions in the following resources. It became increasingly clear to me that, as scientists, we have a moral and social obligation to make sure we are not contributing to racist beliefs about race being grounded in biology or genetics. See this book: <a href="https://link.springer.com/article/10.1007/s11191-014-9697-7"> The Nature of Race: How scientists think and teach about race.</a> and these papers: <a href="https://link.springer.com/article/10.1007/s11191-014-9697-7"> 1. Racial Categories in Medicine: A Failure of Evidence-Based Practice?</a>, <a href="https://link.springer.com/article/10.1007/s11191-014-9697-7"> 1. Racial Categories in Medicine: A Failure of Evidence-Based Practice?</a> <a href="https://www.amazon.co.uk/Reviving-Racial-Medicine-Biomedical-Implications/dp/0954989074"> 2. Reviving Racial Medicine?: The Use of Race/ethnicity in Genetics and Biomedical Research, and the Implications for Science and Healthcare</a>  <a href="https://journals.sagepub.com/doi/10.1177/0306312707083759"> 3. The Standardization of Race and Ethnicity in Biomedical Science Editorials and UK Biobanks</a>
</p>


		<p>4. Evaluating differential gene expression methods for single-cell RNA-sequencing data</p>
		<p>Computing differentially expressed genes (i.e. scoring the confidence with which a gene shows different expression levels between two groups) is challenging for single-cell data. Treating cells as independent observations and estimating differential expression based on a nonparametric rank test, for example, can result in many false positives, where single gene changes are driven by individual patients, who contribute a disproportionate number of cells to the comparison - The observations are not truly independent. A way around this, is to use methods traditionally designed for bulk RNA-sequencing data (which is less sparse than the single-cell equivalent). These approaches use linear models and bayesian statistics to estimate gene changes. We tested an additional approach, where we conducted single-cell comparisons by random sampling of a small number of individuals, followed by aggregation of the differential expression scores. I spent time implementing this method and benchmarked it against "traditional" pseudo-bulking methods to determine which methods to use in our studies.

		<p>5. Evaluating GTEx data heterogeneity for differential gene expression</p>

		<p>The Genotype Tissue Expression (GTEx) Project has quantified gene expression in human tissues from hundreds of
		individuals. During my research internship in the Getz lab, I studied differential expression in these data, for which I evaluated the outputs from two widely-used methods, Limma+Voom and DESeq2. Identifying
		differentially expressed genes in tissues is challenging due to the presence of multiple biological and technical
		confounders, including cellular heterogeneity and batch effects. For this reason, differential expression results
		have not yet been provided as part of official GTEx data releases.
		DESeq2 and Limma+Voom make different statistical assumptions about the distribution of gene expression data
		and, as a result, show vastly different results. To identify biases associated with these packages, I worked with
		randomized GTEx data to empirically test their false discovery rate (FDR). I found the FDR to be consistently higher
		when running DESeq2 as compared to Limma+Voom on data with permuted labels. I also worked with nonrandomized
		GTEx data to characterize the sets of differentially expressed genes called by each package and found
		that both packages call a core set of differentially expressed genes from non-permuted data. This core set of genes
		was consistent across bootstraps and for different sample sizes. While Limma+Voom did not generally make any
		additional calls, DESeq2 consistently called a unique set of genes in addition to the core set. The unique gene sets
		were characterized by lower log-fold changes and higher dispersion estimates and varied for each bootstrapping
		event. These results suggest that DESeq2 is more susceptible to GTEx data sample heterogeneity than
		Limma+Voom, even when controlling for confounders. These results were intended to inform how differential gene expression analysis will be incorporated into the GTEx analysis
		pipeline. </p>

		<div id="writing" class="body-sec3">
		<h4 class="alignleft">writing</h4>
		<p class="alignright"><a href="#top"> back to top </a></p>
		<div style="clear: both;"></div>
	<div class="body-sec2">
		<h3>I enjoy writing and communicating science...</h3>

		<p >1. My colleague Mehdi Jorfi and I wrote this piece on the interplay between microglial states and major risk factors in Alzheimer’s disease, published in the Journal of Neurophysiology, when I first became fascinated by single-cell transcriptomic approaches.</p>
		<p >Abstract: Microglia constitute ~10–20% of glial cells in the adult human brain. They are the resident phagocytic immune cells of the central nervous system and play an integral role as first responders during inflammation. Microglia are commonly classified as “HM” (homeostatic), “M1” (classically activated proinflammatory), or “M2” (alternatively activated). Multiple single-cell RNA-sequencing studies suggest that this discrete classification system does not accurately and fully capture the vast heterogeneity of microglial states in the brain. In fact, a recent single-cell RNA-sequencing study showed that microglia exist along a continuous spectrum of states. This spectrum spans heterogeneous populations of homeostatic and neuropathology-associated microglia in both healthy and Alzheimer’s disease (AD) mouse brains. Major risk factors, such as sex, age, and genes, modulate microglial states, suggesting that shifts along the trajectory might play a causal role in AD pathogenesis. This study provides important insight into the cellular mechanisms of AD and underlines the potential of novel cell-based therapies for AD. <a href="https://pubmed.ncbi.nlm.nih.gov/31365320/"> View the paper.</a></p>
		<p >2. We also compiled a methods protocol describing how to engineer human brain spheroids. </p>
		<p >ABSTRACT: Self-assembling brain spheroids derived from human stem cells closely emulate the tangled connectivity of the human brain, recapitulate aspects of organized tissue structure, and are relatively easy to manipulate compared to other existing three--dimensional (3D) cellular models. However, current platforms generate heterogeneously--sized and short--lived spheroids, which do not robustly and reproducibly model human brain development and diseases. Here, we present a method to generate large--scale arrays of homogeneously--sized 3D brain spheroids derived from human induced pluripotent stem cells (hiPSCs) or immortalized neural progenitor cells to recapitulate Alzheimer’s disease (AD) pathology in vitro. When embedded in extracellular matrix, these brain spheroids develop extensive outward projection of neurites and form networks, which are mediated by thick bundles of dendrites. This array facilitates cost--effective, high--throughput drug screening and mechanistic studies to better understand human brain development and neurodegenerative conditions, such as AD. <a href="https://www.springer.com/gp/book/9781071611739"> View the chapter.</a></p>
		</div>


		<div id="cv" class="body-sec3">
		<h4 class="alignleft">read my cv</h4>
		<p class="alignright"><a href="#top"> back to top </a></p>
		<div style="clear: both;"></div>
	</div>
		<div class="body-sec">
		<a   href="./Djuna_CV.pdf" download>  <p>downnload my CV</p> </a>
		<h3>1. PREVIOUS EDUCATION</h3>
		<h2>PhD Student • Massachusetts Institute of Technology (MIT)</h2>
		<p>09/2019 - present</p>
		<p>MIT Department of Brain and Cognitive Sciences & MIT Computer Science and Artificial Intelligence Lab</p>
		<h2 >Hon B.Sc. • King’s College London, United Kingdom</h2>
		<p>2014 - 2018</p>
		<p>Hon B.Sc. Biomedical Science with Extra Mural Year</p>
		<p>First Class Honors, Overall Score 80% (A+)</p>
		<h2 >Abitur • John F. Kennedy School, Berlin, Germany</h2>
		<p>2001 - 2014</p>
		<p>Abitur, Grade 1.0 (A+)</p>
		<h3>2. AWARDS AND FELLOWSHIPS</h3>
		<h2>Broshy Fellowship</h2>
		<p>2020-2021</p>
		<p>MIT Brain and Cognitive Science departmental award</p>
		<h2>Final Year Biomedical Science Prize for the Best Academic Performance</h2>
		<p>awarded to the best 2 students out of the 200+ students on the Biomedical Science program at King’s College London based on the highest overall degree scores</p>
		<p>2018</p>
		<h2>Mary Clark Travel Award</h2>
		<p> awarded to successful applicants from King’s College London to financially support travel to the U.S. or Commonwealth country during the summer vacation for a study-related project </p>
		<p>2015</p>
		<h3>2. PUBLICATIONS</h3>
		<h2>Peer-Reviewed Articles</h2>
		<p>Kwak SS *, Washicosky KJ *, Brand E, von Maydell D, Aronson J, Kim S, Capen DE, Cetinbas M, Sadreyev R, Ning S, Bylykbashi E, Xia W, Choi SJ, Tanzi RE and Kim DY. (2020) Amyloid-β42/40 ratio drives tau pathology in 3D human neural cell culture models of Alzheimer’s disease. (* = contributed equally). Nat Commun 11, 1377. </p>
		<p>von Maydell D*, Jorfi M.* (2019) The interplay between microglialstates and major risk factors in Alzheimer's disease through the eyes of single- cell RNA-sequencing: beyond black and white. J Neurophysiol 122(4), 1291- 1296. (* = contributed equally).</p>
		<h2>Book Chapter</h2>
		<p>von Maydell, D.; Jorfi, M. A synergistic engineering approach to build human brain spheroids; In Programmed Morphogenesis: Methods and Protocols; Ebrahimkhani, M.; Ed.; Springer Nature, 2020. (in press)</p>
		<h3>3. RESEARCH EXPERIENCE</h3>
		<h2>Research Technician II</h2>
		<p>10/2018 - 07/2019</p>
		<p>MASSACHUSETTS GENERAL HOSPITAL & HARVARD MEDICAL SCHOOL, GENETICS & AGING RESEARCH UNIT</p>
		<h2>Computational Biology Intern </h2>
		<p>08/2018 - 09/2018 </p>
		<p>BROAD INSTITUTE OF HARVARD AND MIT, GTEX GROUP OF THE GETZ LAB</p>
		<h2>Undergraduate Researcher</h2>
		<p>09/2017 – 12/2017</p>
		<p>KINGS COLLEGE LONDON, INSTITUTE OF PHARMACEUTICAL SCIENCE, PHARMACEUTICAL BIOPHYSICS GROUP</p>
		<p>HONORS THESIS: Stratification of Cervico-Vaginal 1H NMR Metabolomics Data Reveals Biomarkers of Spontaneous Preterm Birth, Final mark: 82% (A+) </p>
		<h2>Extra Mural Year Student</h2>
		<p>08/2016 - 07/2017 </p>
		<p>MASSACHUSETTS GENERAL HOSPITAL & HARVARD MEDICAL SCHOOL, GENETICS & AGING RESEARCH UNIT</p>
		<p>EXTRA MURAL YEAR THESIS: Exploring Pathogenic Cascades in Alzheimer’s Disease (AD) by Transcriptomic Analysis of 3D Cultured Human Neurons & Dissecting the Role of Individual Aβ Species on AD Pathogenesis in a 3D Culture Model of AD, Final mark: 90% (A++)</p>
		<h2>Bioinformatics Research Intern</h2>
		<p>07/2015 - 08/2015 </p>
		<p>MASSACHUSETTS GENERAL HOSPITAL & HARVARD MEDICAL SCHOOL, GENETICS & AGING RESEARCH UNIT</p>

		<h3>4. GRADUATE CLASSES</h3>
		<p>- Machine Learning for Genomics / Advanced Computational Biology (present)</p>
		<p>-	Computational Systems Neuroscience Core, PE</p>
		<p>-	Synaptic Mechanisms of Learning and Memory, PE</p>
		<p>-	Quantitative Methods and Computational Models in Neuroscience, A+</p>
		<p>-	Molecular and Cellular Neuroscience, A+</p>
		<p>N.B. PE = reflects performance at any of the levels A, B, or C, under the circumstance of an Institute emergency closure (COVID-19 pandemic) </p>
		</div>


		<div id="teaching" class="body-sec3">
		<h4 class="alignleft">teaching</h4>
		<p class="alignright"><a href="#top"> back to top </a></p>
		<div style="clear: both;"></div>
	</div>

		<div class="body-sec2">
		<h3>I really enjoy teaching and mentoring - In my opinnion it's the best way to learn and extremely gratifying to witness someone else understand a new concept. </h3>
		<p>1. I am currently TAing the graduate class "Quantitative Methods and Computational Models in Neuroscience Teaching", taught by Professor Mehrdad Jazayeri.  We cover linear systems and operations, dimensionality reduction (e.g., PCA), Bayesian approaches, descriptive and generative models, classification and clustering, and dynamical systems. As a whole, the class provides theoretical and practical skills to analyze and model neurobiological data collected at multiple levels in neuroscience: molecular, systems and cognitive levels. </p>
		<p>2. Previously, I was advising an MIT undergraduate researcher in the analysis of single-cell RNA sequencing data, application of statistical techniques to model differentially expressed genes.</p>
		<p>3. I am currently a BCS REF, a trained peer mediator to help other grad students in the BCS department navigate grad school. Check out our website.  <a href=http://bcsrefs.mit.edu/ > View the website</a></p>
		</div>

		<div id="cycling" class="body-sec3">
		<h4 class="alignleft">MIT cycling team</h4>
		<div style="clear: both;"></div>
	</div>

		<div class="body-sec">
		<h3>The MIT cycling team has become my family away from home.</h3>
		 	<p >I started cycling competitively a year before joining MIT, but have become fully devoted to the sport after meeting the awesome people in this team. Have a look at my blog post for the team recounting our fall training camp 2019 experience. <a href="https://cycling.mit.edu/blog/2019/11/11/fall-training-camp-2019/"> View the blog.</a>	If you are considering cycling as a sport (which I HIGHLY recommend, as it is such a fantastic outlet if work is stressful), have a look at some of these links to get you in the mood:</p>
		</div>



				<div id="contact" class="body-sec3">
				<h4 class="alignleft">contact me</h4>

				<div style="clear: both;"></div>
			</div>
				<div class="body-sec2">

				<h3>Reach out if you have questions about my work or would like to discuss an idea for a project!</h3>
				<h3>djuna@mit.edu</h3>
				</div>


	</body
</html>
